test/unit/ui/marker.test.js

```diff
@@ -16,31 +16,60 @@ function createMap() {
 }
 
 test('Marker', (t) => {
+
     t.test('constructor', (t) => {
-        const el = window.document.createElement('div');
-        const marker = new Marker(el);
-        t.ok(marker.getElement(), 'marker element is created');
+        const marker = new Marker();
+        t.ok(marker, 'marker is created');
         t.end();
     });
 
-    t.test('default marker', (t) => {
+    t.test('default marker without element', (t) => {
         const marker = new Marker();
-        t.ok(marker.getElement(), 'default marker is created');
-        t.ok(marker.getOffset().equals(new Point(0, -14)), 'default marker with no offset uses default marker offset');
```

Comment:
we should still test that the offset and anchor of the default marker is as expected